### PR TITLE
chore(release): update release notes [ci skip]

### LINKED ISSUE #2893: Update release notes for 2019.07 release

### PR DESCRIPTION
**Pull request by redallen**
**What**: Update release notes for our upcoming release.

**Additional issues**: Closes #2893

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,3 +1,70 @@
+# 2019.07 release notes (2019-09-09)
```
Comment (mcoker) [inline RELEASE-NOTES.md]:
```suggestion
# 2019.07 release notes (2019-09-10)
```

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,3 +1,75 @@
...
+
+## Components
+- **Accordion:** Allow JSX elements for content and toggle ([#2713](https://github.com/patternfly/patternfly-react/pull/2713))
+- **BackgroundImage:** Fix bug on non-Chrome browsers regarding filter's width ([#2742](https://github.com/patternfly/patternfly-react/pull/2742))
+- **ChipGroup:** Added defaultIsOpen prop to allow chip group to be initially expanded ([#2651](https://github.com/patternfly/patternfly-react/pull/2651))
+- **ClipboardCopy:** Updated if children change ([#2849](https://github.com/patternfly/patternfly-react/pull/2849))
+- **Drawer:** Expose DrawerContent components and remove panelContent prop ([#2733](https://github.com/patternfly/patternfly-react/pull/2733))
+- **Dropdown:**
+  - Added support for dropdown with primary button toggle ([#2857](https://github.com/patternfly/patternfly-react/pull/2857))
+	- Allow custom dropdown items ([#2890](https://github.com/patternfly/patternfly-react/pull/2890))
```
Comment (tlabaj) [inline RELEASE-NOTES.md]: spacing is off over here

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,3 +1,75 @@
...
+	- Added support for toggle icon ([#2748](https://github.com/patternfly/patternfly-react/pull/2748))
+	- Added direction prop ([#2611](https://github.com/patternfly/patternfly-react/pull/2611))
+- **Spinner:** Added new experimental Spinner component! ([#2858](https://github.com/patternfly/patternfly-react/pull/2858))
+- **Switch:** Changed checked property in DOM ([#2749](https://github.com/patternfly/patternfly-react/pull/2749))
+- **Tab:** Allow JSX elements for Tab title ([#2681](https://github.com/patternfly/patternfly-react/pull/2681))
+- **TextArea:** Support defaultValue attribute for uncontrolled TextArea ([#2712](https://github.com/patternfly/patternfly-react/pull/2712))
+- **Toolbar:** Added new experimental Toolbar component! ([#2861](https://github.com/patternfly/patternfly-react/pull/2861))
+- **Tooltip:**
+  - Allow passing properties through to PopoverBase and hence tippy.js ([#2769](https://github.com/patternfly/patternfly-react/pull/2769))
+	- Added the manual trigger option with the isVisible prop ([#2769](https://github.com/patternfly/patternfly-react/pull/2769))
```
Comment (tlabaj) [inline RELEASE-NOTES.md]: spacing off here

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,3 +1,75 @@
...
+- Publish configs as well with icons ([#2701](https://github.com/patternfly/patternfly-react/pull/2701))
+
+## Virtualized Extension
+- Debounce resize event to fix inconsistent grid sizing ([#2754](https://github.com/patternfly/patternfly-react/pull/2754))
+
+## PF3
+- **AsyncTypeAheadSelect:** Forward ref for AsyncTypeAheadSelect ([#2760](https://github.com/patternfly/patternfly-react/pull/2760))
+- **CatalogTile:**
+  - Allow JSX element for CatalogTile's icon ([#2771](https://github.com/patternfly/patternfly-react/pull/2771))
+	- Added iconAlt property to use for alt text for icon image ([#2715](https://github.com/patternfly/patternfly-react/pull/2715))
```
Comment (tlabaj) [inline RELEASE-NOTES.md]: spacing off

**Pull request merged**